news/46.update.rst

```diff
@@ -0,0 +1 @@
+Rework the `tensorflow` hook to improve compatibility across different `tensorflow` versions.
```

Comment:
Can stick some version numbers in here? Unless tensorflow settles down chances are there will be more of these PRs and it makes the changelog useless if users still have to check the source code to see if their tensorflow is supported. Something like;
```suggestion
Fix the ``tensorflow`` hook to be compatible across ``tensorflow`` versions from <1.15.0 up to 2.2.0 (current latest).
```
Also this is reST not markdown so double backticks.

Reply:
Put `[skip-ci]` in the commit message when you do this.

Reply:
Good catch, thanks. 

The latest version is 2.3.0, which I tested on linux with python 3.8.